packages/cli/odahuflow/cli/parsers/local/packaging.py

```diff
@@ -60,6 +62,136 @@ def cleanup_containers():
     cleanup_packaging_docker_containers()
 
 
+def fetch_local_entities(manifest_file, manifest_dir):
+    """
+    Collect entities from manifest files in local FS and return a result
+    Manifests can be collected from file or files inside a directory
+    Manifests from different sources are combined together
+    :param manifest_file: manifest file
+    :param manifest_dir: directory with manifest files
+    :return:
+    """
+
+    entities: List[OdahuflowCloudResourceUpdatePair] = []
+    for file_path in manifest_file:
+        entities.extend(parse_resources_file(file_path).changes)
+
+    for dir_path in manifest_dir:
+        entities.extend(parse_resources_dir(dir_path))
+
+    return entities
+
+
+def parse_entities(
+        entities: List[OdahuflowCloudResourceUpdatePair], pack_id: str
+) -> (ModelPackaging, Dict[str, PackagingIntegration], Dict[str, Connection]):
+    mp: Optional[ModelPackaging] = None
+
+    packagers: Dict[str, PackagingIntegration] = {}
+    connections: Dict[str, Connection] = {}
+
+    for entity in map(lambda x: x.resource, entities):
+        if isinstance(entity, PackagingIntegration):
+            packagers[entity.id] = entity
+        elif isinstance(entity, Connection):
+            connections[entity.id] = entity
+        elif isinstance(entity, ModelPackaging) and entity.id == pack_id:
+            mp = entity
+
+    return mp, packagers, connections
+
+
+def _decode_connection(connection: Connection) -> None:
+
+    encoding = 'utf-8'
```

Comment:
`uft8` is default in Python3 as far as I know, but it is minor, of course.

Reply:
Yes, I know, but prefer that it would be explicit